OctopusDSC/Tests/OctopusDSCHelpers.Tests.ps1

```diff
@@ -67,7 +65,7 @@ Describe "Invoke-OctopusServerCommand" {
     Context "It should not leak passwords" {
 
         BeforeAll {
-            . $script:modulePath
+            # . $script:modulePath
```

Comment:
These comments were removed in a later commit which also included Exe changes so I've left them as is for now.

Reply:
Given we're looking for self-contained PRs (which will become self-contained commits to `master`), we should not intertwine changes between PRs. It's better practice to fix this here.